content/docs/guide-development.md

```diff
@@ -117,7 +117,42 @@ https://renkulab.io/
 
 TODO: explain how to run server
 
+To support the development of workflows in Renku, a set of dedicated funcitonailities, provided as Renku plugins, are made available. In particular those aim to:
 
+* Offer an interactive (and non) visualization of the project Knowledge Graph (`renku-graph-vis` plugin)
+* Create dedicated annotations for the execution of `astroquery` functions and add those to the project Knowledge Graph (`renku-aqs-annotation` plugin)
+
+Those two set of features are provided within different plugins.
+
+### Visualizing project Knowledge Graph with `renku-graph-vis` plugin
```

Comment:
maybe add somewhere how it helps? Like, to refer current development to common ontology and help to choose annotations.

Reply:
with "help to choose annotations", we refer to the annotation that can be added within the notebook right? Then we should also include a link to the dedicated paragraph/document

Reply:
yes, maybe a screenshot which shows how it help to do so, with arrows. else people might wonder why they need this

Reply:
is it correct to mention that the ontology helps in having more information regarding the type of certain entities? I am not sure how to phrase it properly...

Reply:
added some description on the benefits of using the plugins, do they make sense?